src/utilities/routerConfig.js

```diff
@@ -0,0 +1,13 @@
+export const homeURL = '/';

```

Comment:
Replies under your original comments below: 

When I switched to your branch I had some dependency conflicts:
- Added as dependency's.  

I didn't get the same error when attempting to install on main-branch (didn't do so for now). So this may require review later.
- Added as new bug with link to possible fix. 

Would suggest using '.jsx' or '.tsx' (if we end up using typescript) for our components as this gives some better intellisense when coding.
- I have no issue doing this but would love to hear your thoughts / expansion on intelluisense and why .jsx is your preferred choice. 

Small note: As a standard, I use PascalCase for folder names when they contain components. i.e. pages/Contact/Contact.jsx but honestly not necessary just a standard I've seen a lot.
- No issue with this as long as it is consistent throughout the site. 
- As I understand this is just a preference thing. I use kebab case as it clearly differentiates folder from file which (in my head) helps to easily navigate / find / differentiate in large projects.
- Proposal: in situations like this where it is preference and neither of us is wrong, it could be worth defaulting to a specific standard /rule book [(e.g. airbnb)
](https://github.com/airbnb/javascript/tree/master/react)

There's some boilerplate stuff we can still get rid of in the public folder still but can be done later when we update favicon etc.
- Sure thing. My work flow is exactly that. Touch this when I have the assets to replace or at time of production build.

Reply:
Will swap to abort based method for URLs